### PR TITLE
Fix NotFound & NotFoundPage components [Oleksii Savchenko]

### PR DESCRIPTION
**Pull request by Cestrus**
Fixed:
- 100% height,
- add img
- now its main page

Tested components

### CODE REVIEW

**Comment by Cestrus** [comment]
Снимок

**Comment by LordGemma** [comment]
Are you sure that it is a right view? I guess image and text should be in the middle of the page. Also, image should be under the text

File: src/components/NotFound/NotFound.test.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 import { shallow } from 'enzyme';
-import NotFound from './NotFound';
+import NotFound from "./NotFound";
 
-describe('<NotFound/>', () => {
-    it('should has div', () => {
+
+
+describe('<NotFound/> render', () => {
+    it('should render NotFound component', () => {
```
Comment (LordGemma) [inline src/components/NotFound/NotFound.test.js:14]: there are 2 test more. You should check are there image and text?

Comment (Cestrus) [reply src/components/NotFound/NotFound.test.js:14]: ok , i`ll do it

**Comment by Cestrus** [comment]
Снимок

**Comment by LordGemma** [comment]
image

**Comment by AntonBurchak** [comment]
Change approach to inject image. Maybe better use  + h1 and then display it with flexbox

**Comment by Cestrus** [comment]
It was my first decision. But after that i didnt make img chenge its size
as i want :/


ср, 16 дек. 2020 г., 22:16 Anton Burchak <notifications@github.com>:

**Pull request merged**